data/tekton-pipelines/okd/windows10-installer.yaml

```diff
@@ -22,6 +22,7 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: pipeline
+    namespace: kubevirt
```

Comment:
This is added by TTO according to the NS where the pipeline is deployed. Please remove it from here.